.github/workflows/app_deploy_production.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         id: deploy
         with:
           production-branch: master
-          publish-dir: './app/dist'
+          deploy-dir: './app/dist'
```

Comment:
same

Reply:
ah right...

Reply:
yeah, it is wrong: https://github.com/nwtgck/actions-netlify

I went too far with the renaming :sweat_smile: